### DISCUSSION TITLE
Validation set 구성

### DISCUSSION THREAD

**Discussion by jaehwan-AI** [boostcampaitech2/object-detection-level2-cv-06#15] [ideas]
- 인식을 잘 할 것 같은 이미지들을 train set으로 구성한 후 나머지 이미지들을 validation set으로 구성해서 학습 및 검증을 진행
- 진행 후 결과를 바탕으로 train / valid set 재구성

**Comment by ppskj178** [comment]
리더 보드와 유사한 스코어를 출력해주는 셋팅을 찾는게 관건일듯 합니다.

**Reply by jaehwan-AI** [reply]
맞습니다. 가설을 세우면서 LB와 격차가 가장 적은 validation setting을 찾기 위해 실험을 할 것입니다.

**Reply by JiyouSeo** [reply]
리더 보드 제출전에 저희끼리 제출을 결정지을 수 있는 기준(ex 스코어)들을 만드는 것도 괜찮을 것 같습니다.

**Reply by ppskj178** [reply]
추가적으로 제가 mmdetection 라이브러리 보면서 느낀건데
anno_converter 개발을 잘한 것 같아요

분류대회처럼 코드내에서 몇퍼센트 단위로 분리해서 처리하는 형태가 아니라
valid용 json을 따로 생성해서 넣어줘야 되는 방식이더라구요
